### ISSUE TITLE
Some fields are not specific to landsat and should be in separate extensions

### ISSUE THREAD

**Issue by m-mohr** [stac-extensions/sentinel-2#2]
As in https://github.com/stac-extensions/landsat/issues/2, many of the fields should be made more general and be placed in other extensions (many of the percentages, for example).

Generally, I'm planning to talk to ESA to get an official extension in some way, e.g. through an open call proposal. Would you be interested in a collaboration on this?

**Comment by matthewhanson** [comment]
@m-mohr Yes, certainly interested in collaboration here and share your concerns.

Like the Landsat extension, the goal here was to capture the fields that are currently used in the community for Sentinel-2 (which was not using an extension at all, and we needed to document them).   The percentages are really useful to search on but there's no appropriate extension right now.   I think it would be a good new extension so that providers could define their own list of calculated coverage stats.

**Comment by vprivat-ads** [comment]
Hi @matthewhanson this is a great initiative, thank you! Are you working on creating similar extensions for S1 and S3?

**Comment by matthewhanson** [comment]
@vprivat-ads If we end up reindexing/indexing S1/S3 into Earth Search we'd probably create extensions first, but we have no immediate plans to do so.

**Comment by m-mohr** [comment]
ESA would like to discuss which fielda of this extension are important to the users / the STAC community and which we might be able to ditch. Also, we want to discuss whether there are better places for some fields that are not overly specific to the mission.
The document where you can leave comments can be found here: https://eoframework.esa.int/display/CDSE/Copernicus+Data+Space+Ecosystem+%28CDSE%29+STAC+catalogue
Alternatively, leave a comment here and I'll move them over to the Confluence page.
Thanks!
